Source/WebCore/css/StyleProperties.cpp

```diff
@@ -609,24 +640,21 @@ String StyleProperties::fontValue() const
         }
     }
 
-    auto fontSizeString = fontSizeProperty.value()->cssText();
-    auto commonValue = fontSizeString;
+    // This code no longer uses commonValue, for now we define it so we can use appendFontLonghandValueIfExplicit.
```

Comment:
Looks like there are enough callers that don't actually care about the commonValue parameter that it might be nice to add an overload of appendFontLonghandValueIfExplicit that doesn't take one (and either re-implement or just call passing a dummy common value like here).

Reply:
Yes, I expect to make that change, soon, but not in this pull request. The `appendFontLonghandValueIfExplicit` function is only used in two places, `fontValue` and `fontVariantValue`, and the latter has a number of bugs similar to the ones I am fixing here in `fontValue`. I suspect this function will be revised or eliminated entirely when I do that work.